powersimdata/data_access/data_access.py

```diff
@@ -93,6 +93,57 @@ def close(self):
         pass
 
 
+class LocalDataAccess(DataAccess):
+    """Interface to shared data volume"""
+
+    def __init__(self, root=None):
+        self.root = root if root else server_setup.DATA_ROOT_DIR
+
+    def copy_from(self, file_name, from_dir):
+        """Copy a file from data store to userspace.
+
+        :param str file_name: file name to copy.
+        :param str from_dir: data store directory to copy file from.
+        """
+        pass
```

Comment:
Do we need to overwrite the method here. It raises a `NotImplementedError` in the parent class, is that problematic?

Reply:
This does in fact override the behavior - in this case to do nothing, since the directories are symlinked already.